evcxr/src/lib.rs

```diff
@@ -39,3 +39,14 @@ pub use crate::command_context::CommandContext;
 pub use crate::errors::{CompilationError, Error};
 pub use crate::eval_context::{EvalContext, EvalContextOutputs, EvalOutputs};
 pub use crate::runtime::runtime_hook;
+
```

Comment:
Wasn't sure what the best place for this function was. `lib.rs` is mostly just re-exports, but nothing else seemed that relevant, and it didn't seem worth adding a new module just for this function. Let me know if you'd like me to move or change this somehow though.